probe-rs/src/debug/snapshots/probe_rs__debug__debug_info__test__atsamd51p19a__full_unwind.snap

```diff
@@ -214,7 +214,7 @@ expression: stack_frames
     Child Variables:
       name: LocalScopeRoot
       type_name: Unknown
-      value: "<unknown> {\n\tconst_void_pointer: void* = void* @ 0x20000008, \n\tconst_void_const_pointer: const void* = void* @ 0x00001710}"
+      value: "<unknown> {\n\tconst_void_pointer: void* = void* @ 0x20000008,\n\tconst_void_const_pointer: const void* = void* @ 0x00001710}"
```

Comment:
Is there a way to make those strings proper raw multiline strings? that would be sooo much better to read. I am not sure its easily doable with insta though :/

Reply:
https://github.com/mitsuhiko/insta/issues/372